docs/config/build-options.md

```diff
@@ -147,6 +147,12 @@ Set to `false` to disable minification, or specify the minifier to use. The defa
 
 Note the `build.minify` option is not available when using the `'es'` format in lib mode.
 
+Terser must be installed when it is set to `'terser'`.
+
+```sh
+npm add -D terser
```

Comment:
```suggestion
npm install -D terser
```

I believe `npm install` is the more common command, though `npm add` works too as it's an alias.

Reply:
I always use `npm i` and didn't know this alias until recently. But this was intentional to align with #6550.

Reply:
I see. In that case let's make it consistent then. Thanks for checking this.